srcpkgs/libarchive/template

```diff
@@ -59,6 +54,14 @@ bsdtar_package() {
 		mv ${PKGDESTDIR}/usr/share/man/man5/{mtree.5,libarchive-mtree.5}
 	}
 }
+
+bsdunzip_package() {
+	reverts="20160115_2"
```

Comment:
needs _1 too i think

Reply:
Done, but can you elaborate why you think so?

Reply:
because reverts= needs to list every version to revert, and if you don't list 20160115_1 xbps would try to upgrade it to 20160115_2

Reply:
But then wouldn't I have to list every previous version of `bsdunzip`?

Reply:
yes, that's how it works. you're lucky in that bsdunzip was only added and never updated beyond a revbump.

Reply:
Sound fun :D